File: AdotePetJavaProjetct/src/dao/Pessoa.java
import java.util.Date;

public class Pessoa{
    private String nome, cpf, endereco, email, telefone;
    private Date dataNascimento;

		public Pessoa(String nome, String cpf, String endereco, String email, String telefone, Date dataNascimento){
			this.nome=nome;
			this.cpf=cpf;
			this.endereco=endereco;
			this.email=email;
			this.telefone=telefone;
			this.dataNascimento=dataNascimento;
		}

    public String getNome(){
        return nome;
    }
    public void setNome(String nome){
        this.nome=nome;
    }
    public String getCpf(){
        return cpf;
    }
    public void setCpf(String cpf){
        this.cpf=cpf;
    }
    public Date getDataNascimento(){
        return dataNascimento;
    }
    public void setDataNascimento(Date dataNascimento){
        this.dataNascimento=dataNascimento;
    }
    public String getEndereco(){
        return endereco;
    }
    public void setEndereco(String endereco){
        this.endereco=endereco;
    }
    public String getEmail(){
        return email;
    }
    public void setEmail(String email){
        this.email=email;
    }
    public String getTelefone(){
        return telefone;
    }
    public void setTelefone(String telefone){
        this.telefone=telefone;
    }
}

File: AdotePetWeb/devWebLogin/backend/api/model-Eduardo/Pessoa.php
<?php

/**
 * @author Matheus
 */
class Pessoa {
   
    private $nome;
    private $cpf;
    private $dataNascimento;
    private $rua;
    private $bairro;
    private $cidade;
    private $estado;
    private $email;
    private $telefone;
    private $tipoPessoa;
    
    public function get_rua() {
        return $this->rua;
    }

    public function get_bairro() {
        return $this->bairro;
    }

    public function get_cidade() {
        return $this->cidade;
    }

    public function get_estado() {
        return $this->estado;
    }

    public function get_tipoPessoa() {
        return $this->tipoPessoa;
    }

    public function get_cpf() {
        return $this->cpf;
    }

    public function get_nome() {
        return $this->nome;
    }

    public function get_dataNascimento() {
        return $this->dataNascimento;
    }
    public function get_endereco() {
        return $this->endereco;
    }

    public function get_email() {
        return $this->email;
    }

    public function get_telefone() {
        return $this->telefone;
    }
    
    public function set_nome($nome) {
        $this->nome = $nome;
        return $this;
    }

    public function set_cpf($cpf) {
        $this->nome = $cpf;
        return $this;
    }

    public function set_dataNascimento($dataNascimento) {
        $this->dataNascimento = $dataNascimento;
        return $this;
    }

    public function set_endereco($endereco) {
        $this->endereco = $endereco;
        return $this;
    }

    public function set_email($email) {
        $this->email=$email;
        return $this;
    }

    public function set_telefone($telefone) {
        $this->telefone=$telefone;
        return $this;
    }

    public function set_rua($rua) {
        $this->rua=$rua;
        return $this;
    }

    public function set_bairro($bairro) {
        $this->bairro=$bairro;
        return $this;
    }

    public function set_cidade($cidade) {
        $this->cidade=$cidade;
        return $this;
    }

    public function set_estado($estado) {
        $this->estado=$estado;
        return $this;
    }

    public function set_tipoPessoa($tipoPessoa) {
        $this->tipoPessoa=$tipoPessoa;
        return $this;
    }
}